### PR TITLE
Update contributing docs to include `dvctable`

### PR DESCRIPTION
**Pull request by julieg18**
* Updates our contributing docs, adding our custom language `dvctable` to our possible syntax highlighting options.
* See update at https://dvc-org-update-contribu-w0g4ji.herokuapp.com/doc/user-guide/contributing/docs

### CODE REVIEW

**Review by iesahin — COMMENTED** [review]
Thank you @julieg18 LGTM. I think more elaboration for Markdown (saying we use GHF Markdown, can use reference style links etc.) would be better, but for this PR this is all we need.

**Review by rogermparent — APPROVED** [review]

File: content/docs/user-guide/contributing/docs.md [hunk 1]
```diff
@@ -171,11 +171,13 @@ is installed when `yarn` runs (see [dev env](#development-environment)).
...
 - Markdown: Syntax highlighting in fenced code blocks should use the `usage`
-  `dvc`, `yaml`, or `diff` custom languages. `usage` is employed to show the
-  `dvc --help` output for each command reference. `dvc` can be used to show
-  examples of commands and their output in a terminal session. `yaml` is used to
-  show samples of <abbr>DVC files</abbr>, or other YAML contents. `diff` is used
-  mainly for examples of `git diff` output.
+  `dvc`, `dvctable`, `yaml`, or `diff` custom languages. `usage` is employed to
+  show the `dvc --help` output for each command reference. `dvc` can be used to
+  show examples of commands and their output in a terminal session. `dvctable`
+  is used for creating colored, bold, or italic table cells.(You can see an
```
Comment (casperdcl) [inline content/docs/user-guide/contributing/docs.md:177]:
```suggestion
  is used for creating colored, bold, or italic table cells. (You can see an
```

File: content/docs/user-guide/contributing/docs.md [hunk 2]
```diff
@@ -171,11 +171,13 @@ is installed when `yarn` runs (see [dev env](#development-environment)).
...
+  [example](https://dvc.org/doc/start/experiments) of `dvctable` in our "Get
+  Started" section.)`yaml` is used to show samples of <abbr>DVC files</abbr>, or
```
Comment (casperdcl) [inline content/docs/user-guide/contributing/docs.md:179]:
```suggestion
  Started" section.) `yaml` is used to show samples of <abbr>DVC files</abbr>, or
```

File: content/docs/user-guide/contributing/docs.md [hunk 3]
```diff
@@ -171,11 +171,13 @@ is installed when `yarn` runs (see [dev env](#development-environment)).
...
+  other YAML contents. `diff` is used mainly for examples of `git diff` output.
```
Comment (casperdcl) [inline content/docs/user-guide/contributing/docs.md:180]: maybe also nested list for these items?

Comment (julieg18) [reply content/docs/user-guide/contributing/docs.md]: Added nested list for now!

Comment (julieg18) [reply content/docs/user-guide/contributing/docs.md]: Added nested list for now!

**Review by casperdcl — APPROVED** [review]

**Pull request merged**